webapp/channels/src/components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -219,16 +219,24 @@ function isDraftEmpty(draft: PostDraft): boolean {
 }
 
 class AdvancedCreateComment extends React.PureComponent<Props, State> {
+    // public because accessed in advanced_create_comment.test.tsx
+    public draftsForPost: {[postID: string]: PostDraft | null} = {};
+    public doInitialScrollToBottom = false;
+
+    // textBoxRef is accessed in advanced_create_comment.test.tsx
+    private textboxRef: React.RefObject<TextboxClass>;
+    get TextBoxRef() {
+        return this.textboxRef;
+    }
+    set TextBoxRef(newTextBoxRef: any) {
+        this.textboxRef = newTextBoxRef;
+    }
```

Comment:
Like I mentioned in a previous comment about `textBoxRef`, could you leave these as `private readonly` and then use `(instance as any).draftsForPost = ...` in the test file to modify them? Even if it makes the tests a bit awkward, it'd be better to keep these fields private so that the rest of the code in the app is written correctly